vllm/entrypoints/openai/api_server.py

```diff
@@ -672,8 +672,12 @@ async def reset_prefix_cache(raw_request: Request):
         Reset the prefix cache. Note that we currently do not check if the
         prefix cache is successfully reset in the API server.
         """
-        logger.info("Resetting prefix cache...")
-        await engine_client(raw_request).reset_prefix_cache()
+        device = None
+        device_str = raw_request.query_params.get("device")
+        if device_str is not None:
+            device = Device(device_str.upper())
+        logger.info("Resetting prefix cache with specific %s...", str(device))
+        await engine_client(raw_request).reset_prefix_cache(device)
```

Comment:
You cannot take `None` right?

Reply:
Sorry there is a mistake, i corrected L678 from `device = Device(device_str.upper())` to `device = Device[device_str.upper()]`

Reply:
What I actually meant is what if device_str is empty? In this case your device would be None.

Reply:
Sorry for the misunderstand.

If `None` pass to the `CpuGpuBlockAllocator#reset_prefix_cache`

It will reset prefix cache for specified or all devices just like the behavior before.

Reply:
@comaniac Help wanted, why i run it on my local success, but failed in the CI?

Reply:
Not sure. Maybe because your local environment like ruff version or config is different from CI. You can still manually resolve the errors before committing